airflow/opt/providers/etna/etna/get_provider_info.py

```diff
@@ -1,5 +1,10 @@
 # https://airflow.apache.org/docs/apache-airflow-providers/
+from etna.metrics.rollup_metrics import register_rollup_metrics
+
+
 def get_provider_info():
+    register_rollup_metrics()
```

Comment:
Entry point for airflow providers, registers the prometheus metric generator.  This will then "just work" with the existing prometheus-airflow tooling.